ihmc-graphics/src/libgdx/java/us/ihmc/rdx/ui/RDXBaseUI.java

```diff
@@ -102,6 +103,7 @@ public static RDXBaseUI getInstance()
    private final RDX3DPanel primary3DPanel;
    private final ArrayList<RDX3DPanel> additional3DPanels = new ArrayList<>();
    private final RDXVRManager vrManager = new RDXVRManager();
+   private final RDXCapturyManager capturyManager = new RDXCapturyManager();
```

Comment:
This should not go in RDXBaseUI. Instead, RDXCapturyManager should get added to whatever RDX app you need it in.